addons/web/static/src/core/errors/error_service.js

```diff
@@ -106,14 +109,27 @@ export const errorService = {
             if (!originalError && errorsToIgnore.includes(message)) {
                 return;
             }
+            const isRedactedError = !filename && !lineno && !colno;
+            const isThirdPartyScriptError =
+                isRedactedError ||
+                // Firefox doesn't hide details of errors occuring in third-party scripts, check origin explicitly
+                (isBrowserFirefox() && new URL(filename).origin !== window.location.origin);
+            // Don't display error dialogs to public users for third party script errors unless we are in debug mode
+            if (
+                isThirdPartyScriptError &&
+                !(env.services.user && env.services.user.userId) &&
+                !odoo.debug
+            ) {
+                return;
+            }
             let uncaughtError;
-            if (!filename && !lineno && !colno) {
-                uncaughtError = new UncaughtCorsError();
+            if (isRedactedError) {
+                uncaughtError = new UncaughtCorsError(env._t("Third-party script error"));
                 uncaughtError.traceback = env._t(
-                    `Unknown CORS error\n\n` +
-                        `An unknown CORS error occured.\n` +
-                        `The error probably originates from a JavaScript file served from a different origin.\n` +
-                        `(Opening your browser console might give you a hint on the error.)`
+                    `Third-party script error\n\n` +
+                        `An error whose details are inaccessible to the Odoo framework occurred.\n` +
+                        `This error likely comes from a script served from a different origin.\n` +
+                        `Further information may be available in the browser console.`
```

Comment:
I wouldn't have changed the wording "just" to make it more "exact" as it will break translations, and I always prefer to keep a "not perfect" but translated wording than fixing it in stable.
Indeed, people might suddenly now see English messages while they don't speak English.

Eg
![image](https://github.com/odoo/odoo/assets/30048408/cbe4fabc-7905-4eae-840f-544645115f2b)

In some main languages it might be re-translated within a few weeks, even in stable, but for some it will never be (the older the version the truer it gets)

Reply:
Also, it need to be changed in POT if you still want to change that